gson/src/test/java/com/google/gson/JsonObjectTest.java

```diff
@@ -123,27 +136,30 @@ public void testPropertyWithQuotes() {
   /**
    * From issue 227.
    */
-  public void testWritePropertyWithEmptyStringName() {
+  @Test
+  void testWritePropertyWithEmptyStringName() {
     JsonObject jsonObj = new JsonObject();
     jsonObj.add("", new JsonPrimitive(true));
     assertEquals("{\"\":true}", new Gson().toJson(jsonObj));
-
   }
 
-  public void testReadPropertyWithEmptyStringName() {
+  @Test
+  void testReadPropertyWithEmptyStringName() {
     JsonObject jsonObj = JsonParser.parseString("{\"\":true}").getAsJsonObject();
     assertEquals(true, jsonObj.get("").getAsBoolean());
   }
 
-  public void testEqualsOnEmptyObject() {
+  @Test
+  void testEqualsOnEmptyObject() {
     MoreAsserts.assertEqualsAndHashCode(new JsonObject(), new JsonObject());
   }
 
-  public void testEqualsNonEmptyObject() {
+  @Test
+  void testEqualsNonEmptyObject() {
     JsonObject a = new JsonObject();
     JsonObject b = new JsonObject();
 
-    assertEquals(a, a);
+    assertTrue(a.equals(a));
```

Comment:
Changed this because `assertEquals` should not be used to test `equals` implementation (it might return fast without calling `equals` at all).